src/models/refreshToken.ts

```diff
@@ -0,0 +1,17 @@
+import { DataTypes, Model, Sequelize, UUIDV4 } from 'sequelize';
+
+import { CreateActionRefreshToken, IRefreshToken } from '../interfaces';
+
+export interface IRefreshTokenInstance extends Model<IRefreshToken, CreateActionRefreshToken>, IRefreshToken {}
+
+export const getRefreshTokenModel = (sequelize: Sequelize) =>
+    sequelize.define<IRefreshTokenInstance>(
+        'RefreshToken',
+        {
+            id: { type: DataTypes.UUID, defaultValue: UUIDV4, primaryKey: true },
+            token: { type: DataTypes.UUID, defaultValue: UUIDV4 },
```

Comment:
Refresh tokens should also have expiration date for more security

Reply:
@KonanMentor added expiration date for refresh tokens. For now this is just a field with timestamp. I haven't found a possibility to automatically remove data from Sequalize (postgre) by the expiration date, like in mongo